staging_service/autodetect/GenerateMappings.py

```diff
@@ -264,12 +266,12 @@
     apps = mapping[category]
     for app in apps:
         # print("looking at", app)
-        title = app["title"].replace(" ", "_").lower()
+        title = app["title"]
 
         if title not in new_apps:
             # Create a new entry for extensions and id in the app
             app["extensions"] = []
-            app["id"] = counter
+            app["_id"] = counter
             counter += 1
             new_apps[title] = copy.copy(app)
         # Then for the current app we are looking at,
```

Comment:
I think the comments / documentation below need an update, it references the numerical `id` field

Reply:
No, it is correct if you try making the api respond you would see. 
```
curl "https://ci.kbase.us/services/staging_service/importer_mappings/?file_list=none.txt&file_list=file1.gbk"
```